src/App.web.tsx

```diff
@@ -162,7 +163,9 @@ function App() {
                     <I18nProvider>
                       <PortalProvider>
                         <StarterPackProvider>
-                          <InnerApp />
+                          <HiddenRepliesProvider>
```

Comment:
This placement seems a bit wrong to me. Since hidden replies are owned by a user, I think we probably want to move this inside the user-local part keyed by the `did`. Then it would get destroyed between logouts. Does that make sense?